src/CMakeLists.txt

```diff
@@ -2,6 +2,13 @@ if(HAVE_BFD_DISASM)
   set(BFD_DISASM_SRC bfd-disasm.cpp)
 endif()
 
+if (HARDENED_STDLIB)
+  add_compile_definitions("-D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_DEBUG")
+  add_compile_definitions("-D_GLIBCXX_ASSERTIONS")
+  add_compile_definitions("-D_FORTIFY_SOURCE=3")
+  add_compile_options("-O1")
```

Comment:
Not sure forcing `-O1` here is right. This messes with `CMAKE_BUILD_TYPE` and associated CI matrix for it. Seems like `-O1` is required for `_FORTIFY_SOURCE` > 1. Maybe just stick to `=1`?